### DISCUSSION TITLE
SIOP V2 Authentication

### DISCUSSION THREAD

**Discussion by aurelienrouze** [keycloak/keycloak#15932] [Q&A]
Hi everyone, 

After many research, I couldn't find any resource to do an authentication SIOP V2. As I understand, we can only make an authentication with a user. But the principle of Siop V2 is to make an anonymous authentication.

Is there a way to authenticate someone with SIOP V2 protocol ? 

Thanks

Siop V2 resource: https://openid.net/specs/openid-connect-self-issued-v2-1_0.html

**Comment by Voronar** [comment]
As mentioned in keycloak discourse topic [Configuring an Identity Provider with Implicit Flow](https://keycloak.discourse.group/t/configuring-an-identity-provider-with-implicit-flow/5217) keycloak doesn't support nothing except `response_type=code`, but in case of SIOPv2 we need `response_type=id_token`.
